certbot-dns-google/certbot_dns_google/dns_google_test.py

```diff
@@ -74,7 +74,7 @@ def _setUp_client_with_mock(self, zone_request_side_effect):
         rel_path = 'testdata/discovery.json'
         discovery_file = os.path.join(pwd, rel_path)
         http_mock = HttpMock(discovery_file, {'status': '200'})
-        dns_api = discovery.build('dns', 'v1', http=http_mock)
+        dns_api = discovery.build('dns', 'v1', http=http_mock, cache_discovery=False)
```

Comment:
Breaking things on Windows without, because of obscure file locking internal to the google module. In fact it makes test code in sync with tested code, as this parameter is used also here:
https://github.com/certbot/certbot/blob/651de2dd2fea9c79caae96d039f2570518735bdc/certbot-dns-google/certbot_dns_google/dns_google.py#L95-L98